### PR TITLE
Add middle section of the mobile website

### PR DESCRIPTION
**Pull request by ayaderaghul**
- Add 4 cardwork
- Add about me section
- Check linters
- Deploy to netlify
- Update README.md

### CODE REVIEW

File: output/index.html [hunk 1]
```diff
@@ -24,21 +24,159 @@
...
+                </div>
+            </section>
+            <div class="space">
+
             </div>
+            <div class="works">
+            <section class="cardwork">
+                <div class="snapshot">
+                </div>
+                <div class="info">
```
Comment (iambenkis) [inline output/index.html:49]: - [x] Good job creating works class :clap:, that contains projects in your portfolio, I would like to suggest you one thing , You should make sure that each card is centered and make sure that the margin `left and right` is the as as in the Figma design
Your design 
Screenshot from 2023-01-13 16-53-06

Figma design 
Screenshot from 2023-01-13 16-53-59

File: output/index.html [hunk 2]
```diff
@@ -24,21 +24,159 @@
...
+                    <ul class="social-media">
+                        <li class="media-item"><i class="fa-brands fa-twitter"></i></li>
+                        <li class="media-item"><i class="fa-brands fa-linkedin-in"></i></li>
+                        <li class="media-item"><i class="fa-brands fa-github"></i></li>
+                        <li class="media-item"><i class="fa-brands fa-facebook"></i></li>
+                    </ul>
+                </div>
+                <div class="about-button">
+                    Get my resume
+                </div>
```
Comment (iambenkis) [inline output/index.html:158]: - [ ] Kindly adjust this button, so it sticks to the design :+1: 
Your Design

Screenshot from 2023-01-13 16-56-25

Figma Design

Screenshot from 2023-01-13 16-58-01

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ayaderaghul  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AHP15 — APPROVED** [review]
Hi @ayaderaghul , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No linter errors
✅ clean code
✅ Good design
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**